pyspark/bigdl/nn/layer.py

```diff
@@ -902,6 +902,26 @@ def save_graph_topology(self, log_path, bigdl_type="float"):
         callBigDlFunc(bigdl_type, "saveGraphTopology", self.value, log_path)
         return self
 
+    def set_input_formats(self, input_formats, bigdl_type="float"):
+        """
+        set input formats for graph.
```

Comment:
Only support setting input formats for static graph

Reply:
Add check for the class name